lib/Form/Processor/ContentFormProcessor.php

```diff
@@ -186,6 +196,10 @@ public function processCreateDraft(FormActionEvent $event)
         $contentDraft = $this->contentService->createContentDraft($contentInfo, $versionInfo);
         $referrerLocation = $event->getOption('referrerLocation');
 
+        $event->setPayload('contentDraft', $contentDraft);
```

Comment:
How does `contentDraft` relate to `versionInfo`? Is it the same version? If so I'd limit payloads to `content` only.